contrib/poll_stations.py

```diff
@@ -29,10 +29,9 @@
 Outputs something like, which updates regularly:
 
 """
-
+from queue import Queue
```

Comment:
ohh, future provides `queue`... never mind, go back to `import queue`, sorry!

Reply:
Done